### PR TITLE
Add normative references for 3 json schema versions

### LINKED ISSUE #133: Being able to normatively reference JSON Schema
Our largest hurdle for this work item is being able to normatively reference JSON Schema.
JSON Schema has broad usage and industry adoption but isn't a standard in the eyes of the W3C, yet.

The [latest draft of JSON Schema](https://datatracker.ietf.org/doc/draft-bhutton-json-schema/01/) is being standardized in the IETF, though this make take some time, too much time for us to hit our REC goal. We can non-normatively refer to older versions of the spec. But I believe we need to normatively refer to at least one version of JSON Schema (@msporny @OR13 correct me if I'm wrong).

I believe our first step should be reaching out to contacts at the W3C and seeing if we can gain an exception for normatively referencing JSON Schema. I'm not certain how this process works. @iherman can you please advise sensible next steps?

### PR DESCRIPTION
**Pull request by decentralgabe**
fix #133

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/vc-json-schema/pull/170.html" title="Last updated on Jul 5, 2023, 8:26 PM UTC (064c8f9)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vc-json-schema/170/9a34f77...064c8f9.html" title="Last updated on Jul 5, 2023, 8:26 PM UTC (064c8f9)">Diff</a>

### CODE REVIEW

**Comment by decentralgabe** [comment]
Screenshot 2023-07-05 at 1 09 58 PM

File: index.html [hunk 1]
```diff
@@ -339,42 +339,43 @@ <h2>JSON Schema Specifications</h2>
...
-                   <td><a href="https://datatracker.ietf.org/doc/html/draft-bhutton-json-schema-00">Draft 2020-12, draft-bhutton-json-schema-00</a></td>
-                   <td>11 June 2021</td>
+                   <td>[[JSON-SCHEMA-2019-09]]</td>
+                   <td>19 March 2020</td>
+                   <td><a href="https://json-schema.org/draft/2019-09/schema">https://json-schema.org/draft/2019-09/schema</a></td>
                    <td>
-                      • <a href="https://datatracker.ietf.org/doc/html/draft-bhutton-json-schema-00">JSON Schema: A Media Type for Describing JSON Documents</a></br>
-                      • <a href="https://datatracker.ietf.org/doc/html/draft-bhutton-json-schema-validation-00">JSON Schema Validation: A Vocabulary for Structural Validation of JSON</a></br>
-                      • <a href="https://datatracker.ietf.org/doc/html/draft-bhutton-relative-json-pointer-00">Relative JSON Pointers</a>
+                      • <a href="https://datatracker.ietf.org/doc/html/draft-handrews-json-schema-02">JSON Schema: A Media Type for Describing JSON Documents</a></br>
```
Comment (OR13) [inline index.html:362]: not sure about all these IETF refs, prefer to not mix IETF and OpenJS refs for the same standard, I suggest not referring to IETF drafts at all.

**Review by OR13 — CHANGES_REQUESTED** [review]
Remove the IETF refs, just use OpenJS

**Comment by decentralgabe** [comment]
updated 
Screenshot 2023-07-05 at 1 25 25 PM

**Review by OR13 — APPROVED** [review]

**Comment by OR13** [comment]
Given that we will be forced to review this all when we advance to CR, any reason not to merge this now?

**Comment by iherman** [comment]
> Given that we will be forced to review this all when we advance to CR, any reason not to merge this now?

I agree.

Actually: [this blog](https://json-schema.org/blog/posts/future-of-json-schema) on the stability of JSON Schemas is interesting (and important). As we get closer to CR (or even PR) we may have a stable reference that we can rely on once and for all. (I missed this particular blog, my attention was just called on it right now.)

In the meantime, we can simply go on with this PR.

**Pull request merged**